pkg/consts/consts.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package consts
+
+const (
+	RequestHeaderTargetType        = "x-azure-proxy-target"
```

Comment:
could you add a comment here for what this target is?

Reply:
fixed.